gst/cgo_exports.go

```diff
@@ -242,3 +242,34 @@ func goPluginInit(plugin *C.GstPlugin, userData C.gpointer) C.gboolean {
 func goGlobalPluginInit(plugin *C.GstPlugin) C.gboolean {
 	return gboolean(globalPluginInit(wrapPlugin(&glib.Object{GObject: glib.ToGObject(unsafe.Pointer(plugin))})))
 }
+
+//export goLogFunction
+func goLogFunction(
+	_ *C.GstDebugCategory,
+	level C.GstDebugLevel,
+	file *C.gchar,
+	function *C.gchar,
+	line C.gint,
+	object *C.GObject,
+	message *C.GstDebugMessage,
+	_ C.gpointer,
+) {
```

Comment:
the failing GitHub action complains about the redefinition of the `_` param, could that be in this function?

Log:

```
# github.com/go-gst/go-gst/gst
In file included from _cgo_export.c:4:
cgo-gcc-export-header-prolog:68:162: error: redefinition of parameter '_'
cgo-gcc-export-header-prolog:68:45: note: previous declaration is here
_cgo_export.c:440:155: error: redefinition of parameter '_'
_cgo_export.c:440:38: note: previous declaration is here
_cgo_export.c:440:155: warning: omitting the parameter name in a function definition is a C2x extension [-Wc2x-extensions]
Error: Process completed with exit code 1.
```

Reply:
okay that was actually the issue...

Note to self: don't use underscores in functions exported to C